packages/monaco/src/browser/monaco-frontend-module.ts

```diff
@@ -155,7 +155,7 @@ export function createMonacoConfigurationService(container: interfaces.Container
 
     _configuration.getValue = (section, overrides) => {
         const overrideIdentifier = overrides && 'overrideIdentifier' in overrides && overrides['overrideIdentifier'] as string || undefined;
-        const resourceUri = overrides && 'resource' in overrides && !!overrides['resource'] && overrides['resource'].toString();
+        const resourceUri = overrides && 'resource' in overrides && !!overrides['resource'] ? overrides['resource'].toString() : undefined;
```

Comment:
Could you add a declaration on `resourceUri` here specifying that it should be `string | undefined` so that we don't get caught assigning a different type (`false`) to it in the future?